core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponent.scala

```diff
@@ -660,10 +754,17 @@ trait SubmissionComponent {
                  where s.workspace_id=$workspaceId
               """
 
+        def deleteFromSubmissionEntity() =
```

Comment:
I did direct sql here because that's in the rest of this method but should it be a slick query instead?